shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/RpcParamTransformPlugin.java

```diff
@@ -81,14 +81,14 @@ public String named() {
     }
 
     private Mono<Void> formData(final ServerWebExchange exchange, final ServerHttpRequest serverHttpRequest, final ShenyuPluginChain chain) {
-        return Mono.from(serverHttpRequest.getBody()
+        return Mono.from(DataBufferUtils.join(serverHttpRequest.getBody())
```

Comment:
why modify this？

Reply:
formdata may lost data when formdata too large